### ISSUE TITLE
dayClick won't fire on desktops with touch support enabled

### ISSUE THREAD

**Issue by MasterMohsin** [fullcalendar/fullcalendar#3149]
```
$scope.uiConfig = {
      calendar:{
        height: 450,
        editable: true,
        selectable: true,
        header:{
          left: 'title',
          center: '',
          right: 'today prev,next'
        },
        dayClick: $scope.dayClick,
        eventDrop: $scope.alertOnDrop,
        eventResize: $scope.alertOnResize,
        eventClick: $scope.eventClick,
        viewRender: $scope.renderView
      }    
    };
```
Above is my sample code, In this code, 'viewRender' is firing successfully, but 'dayClick' is not working.

**Comment by dancintony** [comment]
I just grabbed the latest update, and I can confirm that dayClick IS firing on my touch devices (tested with iOS and Android), but the jsEvent.pageX and jsEvent.pageY properties are "undefined". Perhaps "$scope.dayClick" is being called, but it's failing because these properties are missing?

**Comment by caseyjhol** [comment]
For pageX and pageY properties, see #3150.

**Comment by pratikdhaboo** [comment]
Guys dayClick is having strange behavior on different machines. I downloaded the latest version 2.7.1 and tried adding dayClick event to one of the demos and it does not work on my browser. For my friend with the same browser version (latest chrome 50.0.2661.102m), it works perfectly.

**Comment by pratikdhaboo** [comment]
Okay i discovered a new bug... on windows devices where touch is enabled, you need to modify line 28-29 in fullcalendar.js to:
`FC.isTouchEnabled = /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent) &&
    !window.__karma__; // workaround for phantomjs falsely reporting touch`

**Comment by caseyjhol** [comment]
Seems related to #3154.

**Comment by arshaw** [comment]
I understand why this is happening. Will get this fixed in next release as soon as I can. There is actually a hidden `isTouch` option in v2.7.1 you can do this...
```
$('#calendar').fullCalendar({
    isTouch: false
});
```

**Comment by arshaw** [comment]
this is fixed in [v2.7.2](https://github.com/fullcalendar/fullcalendar/releases/tag/v2.7.2). could you please verify?

**Comment by dancintony** [comment]
This appears to be fixed for me. I'll do some more testing today to confirm. Thanks again!
